### PR TITLE
Stellarmass

### PR DESCRIPTION
**Pull request by jgducoin**
Finxing bug saying that Smass is not defined when using Sloc.

### CODE REVIEW

File: gwemopt/catalog.py [hunk 1]
```diff
@@ -423,27 +423,27 @@ def get_catalog(params, map_struct):
...
             _2MASS, SDSS = _2MASS[idx], SDSS[idx]
             Lblist = Lblist[idx]
-            mag = mag[idx]
+            magb = magb[idx]
 
         elif params["galaxy_catalog"] == "mangrove":
             GWGC, PGC, HyperLEDA = GWGC[idx], PGC[idx], HyperLEDA[idx]
             _2MASS, SDSS = _2MASS[idx], SDSS[idx]
-            stellarmass = stellarmass[idx]
+            stellarmass. magb = stellarmass[idx], magb[idx]
```
Comment (mcoughlin) [inline gwemopt/catalog.py]: why is this a period?

Comment (jgducoin) [reply gwemopt/catalog.py]: This is a typo indeed. Fixed in next commit.

Comment (mcoughlin) [reply gwemopt/catalog.py]: Could you rebase? David pushed a commit and now there are conflicts.

**Comment by jgducoin** [comment]
Changed included in #57

**Pull request closed without merging**